backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/MetadatenDs.java

```diff
@@ -109,4 +111,32 @@ public Optional<String> getSingleElementSimpleMetadatum(
         .flatMap(
             node -> new Einzelelement(node).getFrameSimpleMetadatum(metadatumSingleElement, date));
   }
+
+  /**
+   * It returns the value for a specific metadata from a single element referenced by its eid for a
+   * specific date.
+   *
+   * @param metadatumSingleElement the enum metadatum
+   * @param eid the eid of the single element
+   * @param date the selected date
+   * @param newValue the new value
+   */
+  public void updateSingleElementSimpleMetadatum(
```

Comment:
Here you have the case missing of removing the parent <einzelelemet> if after removing <artDerNorm> the parent doesn't have any more childs right?